### ISSUE TITLE
Fill and stroke alpha value is ignored in Lottie-web

### ISSUE THREAD

**Issue by karip** [airbnb/lottie-web#1802]
**Tell us about your environment**
Lottie-Preview (Mac/iOS) 3.0.4, Lottie-web 5.5.9.

* **Browser and Browser Version:**
Safari 12.1.2, Firefox 69.0, Chrome 76.

* **After Effects Version:**
N/A

**What did you do? Please explain the steps you took before you encountered the problem.**
1. Open the attached json file in Lottie-web.
2. Observe the results.

**What did you expect to happen?**
There should be three semi-transparent boxes. There should be two semi-transparent strokes. (see the Lottie-Mac/iOS screenshot below). Some of the boxes and strokes have a color which has alpha set to 0.5 and 0.25. I expected Lottie-web to not ignore them.

**What actually happened? Please include as much _relevant_ detail as possible.**
There is only one semi-transparent box. There is only one semi-transparent stroke.

It seems that Lottie-web is ignoring the alpha value in stroke and fill values. Lottie-Mac/iOS renders the alpha values correctly. I have attached screenshots of them below.

![fillstrokealpha-web](https://user-images.githubusercontent.com/6185066/65121502-ff9adf00-d9f7-11e9-92ef-e4cb1889f406.png)
![fillstrokealpha-mac-ios](https://user-images.githubusercontent.com/6185066/65121509-0164a280-d9f8-11e9-9750-89c13f19e491.png)

**Please provide a download link to the After Effects file that demonstrates the problem.**
Here's the json file.

[fillstrokealpha.json.zip](https://github.com/airbnb/lottie-web/files/3624783/fillstrokealpha.json.zip)

**Comment by bodymovin** [comment]
Hi, can you share the .aep?

**Comment by karip** [comment]
Sorry, I don't have an .aep file. It was created in Keyshape.

I have attached another example. I wrote it manually in a text editor, so no .aep file. It might be easier to read than the previous example. I set the blue "Rectangle-alpha" rectangle's alpha value to 0.5.

[fillalpha.json.zip](https://github.com/airbnb/lottie-web/files/3666441/fillalpha.json.zip)

As you can see, Lottie-web ignores the alpha value, but Lottie-Mac/iOS renders it.

![lottie-web](https://user-images.githubusercontent.com/6185066/65828288-ef112100-e2a1-11e9-9828-e11a5344672e.png)
![lottie-ios](https://user-images.githubusercontent.com/6185066/65828290-f0dae480-e2a1-11e9-9c44-35b5cf561081.png)

**Comment by bodymovin** [comment]
Indeed the web player doesn't support opacity as part of the c value, it has to be set in the o property of the same fill.

**Comment by karip** [comment]
Will a future version of the web player support opacity as part of the c value? Or should I try to avoid using that opacity in the c value?

**Comment by bodymovin** [comment]
I don't think it will be supported for now, but maybe in the future I'll take a look at it

**Comment by karip** [comment]
Great! I'll leave this issue open for that.
